### ISSUE TITLE
[NFT Spec] Update spec again

### ISSUE THREAD

**Issue by chihimng** [oursky/likecoin-chain#315]
- royalty config change
- mintable rename
- class fee per byte description

**Comment by chihimng** [comment]
updated. @rickmak please take a look and help update the gist?

i think we can add this to docs website as well, if no more changes are requested from other teams (ref #311)

cc @mitchellchou

**Comment by rickmak** [comment]
For class fee per byte, what is the config we will use to control it? I think it should be at the spec.

And for NFT class and NFT will need to paid the fee for storage. How about royalty config and the royalty distribution?

After a read the royalty spec, I see people can send to any valid address and effective a person can pay a little bit nanolike the spam the chain with infinite account in few transaction. VS now you will need to send nanolike in many transaction to have the same spamming effect. Any concern here?

**Comment by rickmak** [comment]
We should update the rollout plan part too, as we decide with 0.45

**Comment by rickmak** [comment]
gist is updated.

**Comment by elise-ng** [comment]
> For class fee per byte, what is the config we will use to control it? I think it should be at the spec.

it uses the same config as nft fee

> And for NFT class and NFT will need to paid the fee for storage. How about royalty config and the royalty distribution?
After a read the royalty spec, I see people can send to any valid address and effective a person can pay a little bit nanolike the spam the chain with infinite account in few transaction. VS now you will need to send nanolike in many transaction to have the same spamming effect. Any concern here?

I doubt we need extra fee config for bank txn, seems sdk's multisend does not take extra fee for this concern, probably the default fees for accessing kv store is enough.

But sure we could add a max length here to be safe 🤔

**Comment by elise-ng** [comment]
> We should update the rollout plan part too, as we decide with 0.45

yes, and should put it in writing that we want to support authz in next update

**Comment by elise-ng** [comment]
Updated again for finalized release plan.
